oci/spec_opts_linux.go

```diff
@@ -42,13 +42,21 @@ func WithHostDevices(_ context.Context, _ Client, _ *containers.Container, s *Sp
 // If devicePath is a dir it traverses the dir to add all devices in that dir.
 // If devicePath is not a dir, it attempts to add the single device.
 // If containerPath is not set then the device path is used for the container path.
-func WithDevices(devicePath, containerPath, permissions string) SpecOpts {
+func WithDevices(devicePath, containerPath, permissions string, deviceUID, deviceGID uint32) SpecOpts {
```

Comment:
It would be great if we can keep WithDevices() backward-compatible. Can we have another function to pass the UID/GID?

Reply:
Is it used outside containerd? `WithDevices` is at least fairly new (7776e5e) and currently only used by `ctr` and CRI.

Reply:
@kzys @mikebrow I have it now without any API changes or wrappers